pkgs/development/python-modules/hiyapyco/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyyaml
+, jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "HiYaPyCo";
+  version = "0.4.16";
+
+  src = fetchFromGitHub {
+    owner = "zerwes";
+    repo = pname;
+    rev = "release-${version}";
+    sha256 = "1ams9dp05yhgbg6255wrjgchl2mqg0s34d8b8prvql9lsh59s1fj";
+  };
+
+  propagatedBuildInputs = [ pyyaml jinja2 ];
+
+  checkPhase = ''
+    set -e
+    find test -name 'test_*.py' -exec python {} \;
+  '';
```

Comment:
... wish they used a standard test runner like pytest... but i guess this does run tests